stories/atoms/components/button/Button.stories.mdx

```diff
@@ -62,155 +67,204 @@ that will help users to find the action they're looking for.
   )}
 </Story>
 
----
-
-### Properties
+## Properties
 
 <ArgsTable story="Overview" />
 
+## Label
+
+The `label` is the text description of **Button** aswell as its [title](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title).
+If **Button** is an **icon** `variant` the `label` is not shown, however it is still there as a title attribut used for the [tooltip](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tooltip_Role).
```

Comment:
Maybe you could space out the text a bit by skipping a line?
Moreover, I think that the addition of the links on title and tooltip add an additional formatting and I have the impression that the reading is complicated between the text in bold, the code one, plus the links in blue. But this is just my opinion!

Reply:
Fully agree with @lolottetheclash , we need to offer a more readable documentation by limiting the use of multiple text formats.